jena-arq/Grammar/master.jj

```diff
@@ -100,6 +100,38 @@ import org.apache.jena.sparql.core.Quad ;
 public class CLASS extends PARSERBASE
```

Comment:
Looking at the Notes file, there is a comment to run "jj2tokens sparql_11.jj > tokens.txt" to create an initial tokens.txt file, and then manually tidy it up.

Should I re-run jj2tokens, or just manually add the missing entries?

Not sure if that hasn't been used in a while and could generate strange behaviour in other grammars later.